analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.cs

```diff
@@ -330,11 +400,11 @@ public void Sequence(DateTime? value)
             }
             if (!foo.HasValue)
             {
-                Console.WriteLine(foo.Value.ToString());    // FN
+                Console.WriteLine(foo.Value.ToString());    // Noncompliant, foo is empty
```

Comment:
It is needed to test nesting. Actually, this UT is the one that detected an issue in my `ShouldExecute`.

When you concatenate two member access like the ones above you get the following AST:
![image](https://user-images.githubusercontent.com/2676742/225272863-32f4c38c-aa08-4ec5-b471-39b502e8555e.png)

My original implementation of `NullableAccessFinder` didn't call `base.VisitMemberAccessExpression(node);` in the overridden `public override void VisitMemberAccessExpression(MemberAccessExpressionSyntax node)`, and that was producing the false negative, since the `MemberAccessExpressionSyntax` node visit was stopped on the parent, and before reaching the nested `MemberAccessExpressionSyntax`.